app/javascript/app/components/text-area/text-area-styles.scss

```diff
@@ -1,3 +1,7 @@
+@import '~styles/settings';
+
 .textarea {
   font-size: 1rem;
+  border-radius: 4px;
+  border: solid 2.5px $gray3;
```

Comment:
I've created the constant for this type of the border, you can import it from settings.scss:
`$dropdown-border: 2.5px solid $gray3;`